infrastructure/removePublicGCS/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@google-cloud/storage": "*"
```

Comment:
Cloud functions are pretty forgiving because you don't have to worry about having a complicated environment for dependencies, but it's still a good idea to be a little more restrictive here than '*' for versioning. I'd suggest setting this to 'at least version I tested with' and 'less than next API breaking version', like so:
>=1.5.2 <2

Reply:
GitHub ate my damn comment, >=1.5.2 <2